Icon Update with new color system

edgalindo opened the pull request:
Color system was updated.

Commit:
Icon Update with new color system

Commit:
Icons and Vignetts in new color system

Update of Icons and Vignetts to new color system. Icons were placed in a corect position within a width 208 and height 116.

dnsos reviewed (changes requested):
Hey @edgalindo, thanks for the new assets!

I noticed two things:

1. the arrow icon in the breadcrumbs has a different size. It should be 21x21px.
2. the icon for the step-by-step explanation next to the iframes is too small. Should be 32x32px.

Could you update this? Everything else seems fine.

Commit:
Update Icon Sizes

Replaced the Icon size for the breadcrumb arrow (off and on) and instructions respectivelygebrauchsanleitung

Commit:
Rezise of breacrumb arrow

Rechanged the Icon size of arrow_crumb within the surface area of 21x21 pxl

edgalindo commented:
@dnsos Question to arrow breadcrumb:

resized the arrow icon in the breadcrumbs to 15x15 pxl within 21x21px (artwork area), but appears to big on the live version  
Could you look at this? Do you need to change the styles in SAAS, or should I still change the arrow size?

Commit:
New Icons for Buttons on 2. level

Redesign of 2. Level (formely a list points) with icons for buttons to different ID services

Commit:
New Icons for Buttons on 2. level "Passport"

Redesign of 2. Level (formely a list points) with icons for buttons to different Passport services

Commit:
New Icons for Buttons on 2. level "App"

Redesign of 2. Level (formely a list points) with icons for buttons to different Appartment services

Commit:
New Icons for Buttons on 2. level "kita"

Redesign of 2. Level (formely a list points) with icons for buttons to different daycare (kita) services

Commit:
Udate Icons and Vignettes

Updating Color system for Vignettes and Upload of new graphic components for the 3. level.

Commit:
New Icon and graphic components

Icon update and new graphic components for 3. level

Commit:
New Graphic components

Graphic component for. Checkbox and Tooltip window for 3. level

dnsos committed:
style: update breadcrumb icon asset

dnsos reviewed (approved):
Hey @edgalindo, sorry for taking care of this only now. I updated the breadcrumb icon, now it should be the correct size.

~If all other changes are included now, please let me know. Then I would merge this.~

Everything's included now, so this will be merged!
